tests/sla_tests.js

```diff
@@ -429,4 +465,180 @@ describe('SLA Bridge and Channels Tester', function() {
       }, asyncDelay);
     } 
   });
+  it('should mark a device as RINGING when dialing an outbound channel',
+      function(done) {
+    var client = getMockClient();
+    var inbound = getMockChannel();
+    inbound.inbound = true;
```

Comment:
From a naming perspective, ```inbound.inbound``` is pretty weird. I'd name the channel something other than ```inbound``` to avoid the duplication.